### PR TITLE
Allow ignore findfiles

### PR DESCRIPTION
**Pull request by hanxiaop**
For https://github.com/istio/tools/pull/2751 to exclude releasenote template.

### CODE REVIEW

**Review by jwendell — COMMENTED** [review]
Is it that hard to make these templates pass the linter? Is there a way to bypass the linter with some special comment like go linter has `//nolint` ?

File: files/common/Makefile.common.mk [hunk 1]
```diff
@@ -19,7 +19,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FINDFILES=find . \( -path ./common-protos -o -path ./.git -o -path ./out -o -path ./.github -o -path ./licenses -o -path ./vendor \) -prune -o -type f
+ifndef FINDFILES
```
Comment (dhawton) [inline files/common/Makefile.common.mk:22]: Why not use ```FINDFILES ?=``` instead of the conditional?

**Comment by ericvn** [comment]
I would be inclined to simply add the `./cmd/gen-release-notes/templates` to the list of paths to ignore. Some of the paths listed are repo specific. To be more elaborate, you could specify an additional variable in the Makefile.core.mk like `MARKDOWN_LINT_IGNORE=./cmd/gen-release-notes/templates` and then here add that variable into the command.

**Comment by hanxiaop** [comment]
> Is it that hard to make these templates pass the linter? Is there a way to bypass the linter with some special comment like go linter has `//nolint` ?

I don’t think the markdown lint tool has similar features. Maybe there's an ignore file can be specified but it will not be repo-specific. I think @ericvn 's suggestion of FINDFILES_IGNORE makes sense.

**Review by ericvn — APPROVED** [review]

**Comment by ericvn** [comment]
Thanks

**Pull request merged**